intel/geoip/location.go

```diff
@@ -53,29 +60,30 @@ func (l *Location) EstimateNetworkProximity(to *Location) (proximity int) {
 
 		Weighting Configuration
 
-		- Continent match: 30
-		- Country match: 25
-		- ASOrg match: 20
-		- ASN match: 15
-		- Coordinate distance: 0-10
+		- Continent match: 25
+		- Country match: 20
+		- ASOrg match: 15
+		- ASN match: 10
+		- Coordinate distance: 0-30
 	*/
```

Comment:
just remove that comment and point to the `const` section above. This will for sure be wrong in a few weeks/months again